src/blame.h

```diff
@@ -84,6 +84,8 @@ struct git_blame {
 	int num_lines;
 	const char *final_buf;
 	git_off_t final_buf_size;
+
+	git_mailmap *mailmap;
```

Comment:
You should probably move this just after `repository` to avoid unnecessary padding